### PR TITLE
Calculates free ram with more robust /proc/meminfo instead of `free`

### PR DESCRIPTION
**Pull request by loosecannon93**

### CODE REVIEW

**Comment by loosecannon93** [comment]
@yuyantingzero

File: perfkitbenchmarker/linux_benchmarks/hpcc_benchmark.py [hunk 1]
```diff
@@ -106,7 +106,7 @@ def CreateHpccinf(vm, benchmark_spec):
   if FLAGS.memory_size_mb:
```
Comment (yuyantingzero) [inline perfkitbenchmarker/linux_benchmarks/hpcc_benchmark.py]: Can you invoke DropCaches before grabbing the free memsize?
https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/6e0e86d073aa9567b704a8b452970d5826c6fbfb/perfkitbenchmarker/linux_virtual_machine.py#L493

**Comment by yuyantingzero** [comment]
LGTM, thanks for the fix!

**Comment by peterfeiner** [comment]
This isn't quite right.  You want MemFree + Buffers + Cached. For example, on my 14.04 machine:

```
$ free
             total       used       free     shared    buffers     cached
Mem:      65901572   62107028    3794544     313432    1412824   45675300
-/+ buffers/cache:   15018904   50882668
Swap:     67031036     601520   66429516
$ free | sed -n 3p | awk {'print $4'}
50895300
$ egrep '^(MemFree|Buffers|Cached):' /proc/meminfo
MemFree:         3804428 kB
Buffers:         1412832 kB
Cached:         45675252 kB
$ echo $((3804428+1412832+45675252))
50892512
```

**Pull request merged**